.github/workflows/renovate_config_validation.yaml

```diff
@@ -0,0 +1,22 @@
+name: Renovate Config validation
+permissions:
+  contents: read
+on:
+  workflow_dispatch:
+  pull_request:
+    types: [ opened, reopened, synchronize ]
+    branches:
+      - main
+    paths:
+      - .github/workflows/renovate_config_validation.yaml
+      - renovate.json
+
+jobs:
+  renovate-config-validation:
+    runs-on: ubuntu-latest
+    name: Validate the Renovate configuration
+    steps:
+      - name: Validate the Renovate JSON
+        run: |
+          set -euxo pipefail
+          npx --yes --package renovate -- renovate-config-validator
```

Comment:
Didn't know that renovate actually provides something that does a more enhanced validation than just checking if the JSON fits to the schema. Well played @silvestre 🤝 👍, love it :D, I definitely learned something here 🤓